nixos/tests/docker.nix

```diff
@@ -11,6 +11,7 @@ import ./make-test.nix ({ pkgs, ...} : {
       { config, pkgs, ... }:
         {
           virtualisation.docker.enable = true;
+          virtualisation.docker.extraOptions = "--storage-driver=overlay";
```

Comment:
Shouldn't this be in the nixos service instead? If the test doesn't work without it, I wonder if the normal system works without it...